types/meteor/mongo.d.ts

```diff
@@ -243,3 +267,15 @@ declare module "meteor/mongo" {
         }
     }
 }
+
+declare module MongoInternals {
+    interface Connection {
+        db: MongoDb;
```

Comment:
Could we also add `client` here?

Reply:
What'd I set it to? `MongoDb` also?

Reply:
No, [`MongoClient`](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/882f5542a4f0f9793f944f3650adec7e7ee98fe3/types/mongodb/index.d.ts#L51-L53).